src/sections/Community/Handbook/repository.js

```diff
@@ -204,6 +209,64 @@ const Repository = () => {
                 </div>
               );
             })}
+            <a id="Maintainers">
+              <h2>Maintainers</h2>
+            </a>
+
+            {Maintainers.map((Maintainers, index) => {
```

Comment:
maintainer  singular done

No, It isnt possible without map as each project is mapped with its respective maintainers individually and hence can't be accessed directly.

Reply:
I was referring to the parent `Maintainers.map` and not the one inside it. It can be done, but we can leave it as it is for a better coding convention.